### ISSUE TITLE
Cluster moved to un-managed state alert

### ISSUE THREAD

**Issue by ltrilety** [Tendrl/notifier#157]
After un-manage and import of cluster I found an alert which is saying that cluster goes to un-manage state. It's not true as the cluster is imported again.

![unmanage-alert](https://user-images.githubusercontent.com/22369834/36421858-61ba5180-163a-11e8-9e93-c00c70af9771.png)

Tested version:
tendrl-commons-1.6.0-1.el7.centos.noarch
tendrl-api-1.6.0-1.el7.centos.noarch
tendrl-ui-1.6.0-1.el7.centos.noarch
tendrl-grafana-selinux-1.5.4-2.el7.centos.noarch
tendrl-ansible-1.5.5-1.el7.centos.noarch
tendrl-notifier-1.6.0-1.el7.centos.noarch
tendrl-node-agent-1.6.0-1.el7.centos.noarch
tendrl-api-httpd-1.6.0-1.el7.centos.noarch
tendrl-selinux-1.5.4-2.el7.centos.noarch
tendrl-grafana-plugins-1.6.0-1.el7.centos.noarch
tendrl-monitoring-integration-1.6.0-1.el7.centos.noarch

**Comment by shtripat** [comment]
@ltrilety do you mean this notification should vanish once cluster is imported again? Actually there is no clearing events for this so it remains there.
@r0h4n @julienlim @nthomas-redhat commnets?

**Comment by nthomas-redhat** [comment]
@shtripat , I don't think this is an alert condtion. This can be just logged as an event.
For alert conditions we need to have a corresponding clearing event.

**Comment by shtripat** [comment]
@nthomas-redhat agree. Lets hear from others as well on this and based on that take the action.

**Comment by ltrilety** [comment]
@shtripat @nthomas-redhat I agree that it should not be alert at all. Just an event saying that cluster was un-managed in past and where to look for it's data.

**Comment by julienlim** [comment]
@ltrilety @nthomas-redhat @r0h4n @shtripat 
If a cluster goes into a different state of managed / unmanaged, it should be a logged as an event (which I'm referring to as a logged entry).  If a cluster is unmanaged, and we raise an alert (to get users attention) to indicate it's unmanaged, when it's later managed again, it definitely will need the corresponding alert to show it's cleared.

Now the question I think is whether or not an alert is needed.  I think we probably don't need to do so as (1) there probably are not many storage administrator users on at the same time managing the same cluster, and (2) the action of manage/unmanage is user-initiated so the user will typically assume it happens once the user initiates the action.  What the user expects will be alerts to get his/her attention if the action fails, e.g. unmanage or manage/import fails (and clearing when the failure is no longer presented).

**Comment by shtripat** [comment]
@julienlim regarding failure of import/un-manage, the task itself would be failed and we already indicate an import failure for a cluster in cluster-list view.

**Comment by nthomas-redhat** [comment]
@julienlim ,I agree with @shtripat. Its a user action and details of failure/success is presented as part of the task itself. Logging an event is good enough is what I am think right now.
Also if we start adding the alerts for all sort of user actions, it will flood the admins inbox which they may not like.
Make sense?

**Comment by julienlim** [comment]
@nthomas-redhat @shtripat 

I'm in agreement that we don't need to raise an alert for the manage/unmanage as I already stated so above ("Now the question I think is whether or not an alert is needed.  I think we probably don't need to do so  as....").

**Comment by shtripat** [comment]
Ack. Sending a PR to just log and event with archive location now.
